docs/architecture/adr-020-protobuf-client-encoding.md

```diff
@@ -0,0 +1,172 @@
+# ADR 020: Protocol Buffer Client Encoding
+
+## Changelog
+
+- 2020 March 06: Initial Draft
+
+## Status
+
+Proposed
+
+## Context
+
+This ADR is a continuation of the motivation, design, and context established in
+[ADR 019](./adr-019-protobuf-state-encoding.md), namely, we aim to design the
+Protocol Buffer migration path for the client-side of the Cosmos SDK.
+
+Specifically, the client-side migration path primarily includes tx generation and
+signing, message construction and routing, in addition to CLI & REST handlers and
+business logic (i.e. queriers).
```

Comment:
Can we leave queriers to a separate ADR as I was previously suggesting? There is other stuff to address there like JSON, maybe grpc, etc.

Reply:
Added more context about querying and how it'll be addressed in a future ADR.